pep-0695.rst

```diff
@@ -382,9 +390,9 @@ includes the type parameters. Type parameters can be accessed by name
 within inner scopes. As with other symbols in Python, an inner scope can
 define its own symbol that overrides an outer-scope symbol of the same name.
 
-Type parameters declared earlier in a type parameter list are visible to
-type parameters declared later in the list. This allows later type parameters
-to use earlier type parameters within their definition. While there is currently
+Type parameters are visible to other
+type parameters declared elsewhere in the list. This allows type parameters
```

Comment:
"Are visible to" was slightly unclear language to me -- maybe "can be referred to by"? Or something like that?

Reply:
I think this is standard nomenclature for scopes. For example, https://docs.python.org/3.12/reference/executionmodel.html#resolution-of-names uses it.

Reply:
*Grumbles, mutters darkly*

...Fine